velox/functions/sparksql/tests/UuidTest.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/sparksql/tests/SparkFunctionBaseTest.h"
+
+namespace facebook::velox::functions::sparksql::test {
+namespace {
+
+class UuidTest : public SparkFunctionBaseTest {
+ protected:
+  std::optional<std::string> uuid(int64_t seed, int32_t partitionIndex = 0) {
+    setSparkPartitionId(partitionIndex);
+    return evaluateOnce<std::string>(
+        fmt::format("uuid({})", seed), makeRowVector(ROW({}), 1));
+  }
+
+  VectorPtr
+  randWithBatchInput(int64_t seed, int32_t partitionIndex, int32_t batchSize) {
```

Comment:
consistency: uuid and randWithBatchInput are similar in logic, but their names are very different

consider renaming randWithBatchInput to uuid 

or, use uuidOne and uuidMany names